gcloud/pubsub/client.py

```diff
@@ -65,20 +65,12 @@ def list_topics(self, page_size=None, page_token=None):
                   more topics can be retrieved with another call (pass that
                   value as ``page_token``).
         """
-        params = {}
-
-        if page_size is not None:
-            params['pageSize'] = page_size
-
-        if page_token is not None:
-            params['pageToken'] = page_token
-
-        path = '/projects/%s/topics' % (self.project,)
-        resp = self.connection.api_request(method='GET', path=path,
-                                           query_params=params)
+        conn = self.connection
+        resources, next_token = conn.list_topics(
```

Comment:
I hadn't really decided, yet.  If we mean to keep supporting both JSON-over-REST and proto-over-gRPC, then the latter seems most sensible (even though it loses some of the performance benefits of protobufs).  The deserialization can at least be done via `google.protobuf.json_format.Parse`.